File: src/main/java/kitchenpos/menus/tobe/infra/ToBeJpaMenuGroupRepository.java
package kitchenpos.menus.tobe.infra;

import kitchenpos.menus.domain.MenuGroup;
import kitchenpos.menus.domain.MenuGroupRepository;
import kitchenpos.menus.tobe.domain.ToBeMenuGroup;
import kitchenpos.menus.tobe.domain.ToBeMenuGroupRepository;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface ToBeJpaMenuGroupRepository extends ToBeMenuGroupRepository, JpaRepository<ToBeMenuGroup, UUID> {
}

File: src/main/kotlin/kitchenpos/menu/tobe/infra/TobeJpaMenuGroupRepository.kt
package kitchenpos.menu.tobe.infra

import java.util.*
import kitchenpos.menu.tobe.domain.MenuGroup
import kitchenpos.menu.tobe.domain.MenuGroupRepository
import org.springframework.data.jpa.repository.JpaRepository

interface TobeJpaMenuGroupRepository : MenuGroupRepository, JpaRepository<MenuGroup, UUID> {

}
